### PR TITLE
update: remove unsupported .NET Core versions

### PR DESCRIPTION
**Pull request by brnhensley**
We dropped support of agent v9 last week and all versions of .NET Core below 3.1. I've requested the .NET team review this for mistakes

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx [hunk 1]
```diff
@@ -49,10 +49,12 @@ Before you install the New Relic .NET agent on Windows or Linux, make sure your
     * [.NET Core version 3.1](https://devblogs.microsoft.com/dotnet/net-core-3-1-will-reach-end-of-support-on-december-13-2022/) on December 13, 2022
     * [.NET version 5](https://devblogs.microsoft.com/dotnet/dotnet-5-end-of-support-update/) on May 10, 2022
```
Comment (tippmar-nr) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx:50]: .NET 7 is also EOL as of May 14, 2024.

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx [hunk 2]
```diff
@@ -49,10 +49,12 @@ Before you install the New Relic .NET agent on Windows or Linux, make sure your
...
 
-    The official product lifecycle start and end dates can be found in the [Microsoft documentation](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core). Our .NET agent support of these framework versions end with the latest 9.x New Relic .NET agent. Starting from the New Relic .NET agent version 10.0, we will target .NET Core 3.1 onwards.
+    The official product lifecycle start and end dates can be found in the [Microsoft documentation](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core). 
+
+    Our .NET agent support of Core version below 3.1 ended with our support of 9.x New Relic .NET agent. Starting from the New Relic .NET agent version 10.0, we will target .NET Core 3.1 onwards.
   </Callout>
 
-    The .NET agent supports .NET Core versions 2.0, 2.1, 2.2, 3.0, 3.1, and .NET 5.0, 6.0, 7.0, and 8.0.
+    The .NET agent supports .NET Core version 3.1, and .NET 5.0, 6.0, 7.0, and 8.0.
```
Comment (tippmar-nr) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx:57]: Remove 7.0 from this list.

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx [hunk 3]
```diff
@@ -159,35 +121,17 @@ Before you install the New Relic .NET agent on Windows or Linux, make sure your
...
     id="target-framework"
     title="Target framework version"
   >
-    The .NET agent only supports applications targeting .NET Core 2.0, 2.1, 2.2, 3.0, 3.1, and .NET 5.0, 6.0, 7.0, and 8.0. You can find the target framework in your `.csproj` file. Agent compatibility varies across different versions of .NET Core. For information about which agent version to use for your application's targeted version of .NET Core, see the above section of our documentation, <DoNotTranslate>**[Microsoft .NET Core version](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core/#net-version)**</DoNotTranslate>.
+    The .NET agent only supports applications targeting .NET Core 3.1, and .NET 5.0, 6.0, 7.0, and 8.0. You can find the target framework in your `.csproj` file. Agent compatibility varies across different versions of .NET Core. For information about which agent version to use for your application's targeted version of .NET Core, see the above section of our documentation, <DoNotTranslate>**[Microsoft .NET Core version](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core/#net-version)**</DoNotTranslate>.
```
Comment (tippmar-nr) [inline src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx:131]: Remove 7.0

**Review by homelessbirds — APPROVED** [review]

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx (hunk 3 above)
Comment (homelessbirds) [reply src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx]: Fixed!

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx]: Fixed!

File: src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core.mdx:50]: Fixed!

**Pull request merged**